x/perpetual/keeper/mtp_trigger.go

```diff
@@ -12,7 +12,7 @@ import (
 // 1. Settle funding fee, if unable to settle whole, close the position
 // 2. Settle interest payments, in unable to pay whole, close the position
 // 3. Update position health and check if above minimum threshold, if equal or lower close the position
-func (k Keeper) MTPTriggerChecksAndUpdates(ctx sdk.Context, mtp *types.MTP, pool *types.Pool, ammPool *ammtypes.Pool) (repayAmt, returnAmt, fundingFeeAmt, fundingAmtDistributed, interestAmt, insuranceAmt math.Int, allInterestsPaid, forceClosed bool, perpetualFeesCoins types.PerpetualFees, err error) {
+func (k Keeper) MTPTriggerChecksAndUpdates(ctx sdk.Context, mtp *types.MTP, pool *types.Pool, ammPool *ammtypes.Pool) (repayAmt, returnAmt, fundingFeeAmt, fundingAmtDistributed, interestAmt, insuranceAmt math.Int, allInterestsPaid, forceClosed bool, perpetualFeesCoins types.PerpetualFees, closingPrice math.LegacyDec, err error) {
```

Comment:
Should we rename closingPrice as closingPriceIfForceClosed or something like? Otherwise, it will return nil value with a nil error. Or maybe we can add a comment?